pages/common/rg.md

```diff
@@ -0,0 +1,19 @@
+# ripgrep
+
+> A fast commandline search tool.
+
+- Recursively search the current directory for a regex pattern:
+
+`rg {{pattern}}`
+
+- Search in all .gitignored and hidden files:
+
+`rg -uu {{pattern}}`
+
+- Search only in html or css files for a pattern:
+
+`rg -thtml -tcss {{pattern}}`
```

Comment:
Does the `-t` correspond to an actual word? I suppose "file **type**"? If so, it should be mentioned in the description. Also, if spaces are allowed after the `-t`, it would be nicer to include them, for readability.